app/views/shared/summary_cards/_list.html.slim

```diff
@@ -4,7 +4,7 @@
       div class="entity-icon entity-icon--lists"
```

Comment:
@ptrikutam Indeed, I forgot to include the partial showing the tags in the list summary card! Thanks for catching that.